app/__init__.py

```diff
@@ -5,12 +5,14 @@
 from flask_limiter import Limiter
 from flask_limiter.util import get_remote_address
 
-
 db = SQLAlchemy()
 migrate = Migrate()
 
 API_VERSION = "1.0"
 
+# healthcheck uses API_VERSION so this has to come after it's declared
+from app.healthcheck import add_health_check # noqa
```

Comment:
I thought about having all the logic in `healtcheck.py` directly in this file, but it seemed like it should go in its own file. But I had to do this gross thing to make it work while still returning the API_VERSION. Open to suggestions on something different to do about this

Reply:
Yeah the circular import is great. 

honestly we could probably declare API version in the CI build instead. And then inject environment variables. So then each one wouldn't have to reference the other.